### PR TITLE
release action update

### PR DESCRIPTION
**Pull request by ovidiuolteanu**
Release update for latest reproducible build flow.

### CODE REVIEW

**Review by andreibancioiu — APPROVED** [review]

**Review by andreibancioiu — COMMENTED** [review]
Also add one for pull request (as pre-checks):

https://github.com/multiversx/mx-reproducible-contract-build-example-sc/blob/main/.github/workflows/on_pull_request_build_contracts.yml

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -1,86 +1,15 @@
...
-            echo " - **${filename}**: \`${checksum}\`" >> notes.txt
-          done
-          gh release create ${{ github.event.inputs.tag }} --target=$GITHUB_SHA --prerelease --title="${{ github.event.inputs.title }}" --generate-notes --notes-file=notes.txt
-          sleep 10
-          gh release upload ${{ github.event.inputs.tag }} $(find ./assets -type f)
\ No newline at end of file
+    uses: multiversx/mx-sc-actions/.github/workflows/reproducible-build.yml@v2.3.1
+    with:
+      image_tag: v5.1.0
+      attach_to_existing_release: true
```
Comment (andreibancioiu) [inline .github/workflows/release.yml:15]: Missing empty line at end of file.

Comment (ovidiuolteanu) [reply .github/workflows/release.yml]: Done.

**Comment by ovidiuolteanu** [comment]
> Also add one for pull request (as pre-checks):
> 
> https://github.com/multiversx/mx-reproducible-contract-build-example-sc/blob/main/.github/workflows/on_pull_request_build_contracts.yml

For pull request we will keep the existing action as it is more relevant for now to just run the tests over there.

**Review by andreibancioiu — APPROVED** [review]

**Pull request merged**